### PR TITLE
Some updates about how to install solidity.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/installing-solidity.rst [hunk 1]
```diff
@@ -35,13 +35,14 @@ npm / Node.js
 =============
 
 Use `npm` for a convenient and portable way to install `solcjs`, a Solidity compiler. The
-`solcjs` program has fewer features than all options further down this page. Our
+`solcjs` program has fewer features than the other options further down this page. Our
```
Comment (ekpyron) [inline docs/installing-solidity.rst]: Maybe just `` `solcjs` (has fewer features and) does not support all the options described below.``? "fewer features than the other options" sounds a bit strange.

File: docs/installing-solidity.rst [hunk 2]
```diff
@@ -35,13 +35,14 @@ npm / Node.js
...
 :ref:`commandline-compiler` documentation assumes you are using
-the full-featured compiler, `solc`. So if you install `solcjs` from `npm` then you will
-stop reading the documentation here and then continue to `solc-js <https://github.com/ethereum/solc-js>`_.
+the full-featured compiler, `solc`. The usage of `solcjs` is documented inside own
```
Comment (ekpyron) [inline docs/installing-solidity.rst]: ``is documented inside its own repository``

File: docs/installing-solidity.rst [hunk 3]
```diff
@@ -99,7 +101,8 @@ We are also releasing a `snap package <https://snapcraft.io/>`_, which is instal
 
     sudo snap install solc
 
-Or if you want to help testing the unstable solc with the most recent changes from the development branch:
+If you want to help testing the unstable solc with the most recent changes
```
Comment (ekpyron) [inline docs/installing-solidity.rst]: ``the unstable version of solc`` or ``the current development version of solc``?

File: docs/installing-solidity.rst [hunk 4]
```diff
@@ -240,9 +241,8 @@ in Visual Studio 2017 Build Tools or Visual Studio 2017:
 External Dependencies
 ---------------------
 
-We now have a "one button" script which installs all required external dependencies
-on macOS, Windows and on numerous Linux distros.  This used to be a multi-step
-manual process, but is now a one-liner:
+We have a "one button" script which installs all required external dependencies
```
Comment (ekpyron) [inline docs/installing-solidity.rst]: Maybe just a ``helper script``?

File: docs/installing-solidity.rst [hunk 5]
```diff
@@ -329,7 +331,7 @@ The Solidity version string contains four parts:
 - the version number
 - pre-release tag, usually set to ``develop.YYYY.MM.DD`` or ``nightly.YYYY.MM.DD``
 - commit in the format of ``commit.GITHASH``
-- platform has arbitrary number of items, containing details about the platform and compiler
+- platform, which has arbitrary number of items, containing details about the platform and compiler
```
Comment (ekpyron) [inline docs/installing-solidity.rst]: ``which has an arbitrary number of items``

File: docs/installing-solidity.rst (hunk 1 above)
Comment (chriseth) [reply docs/installing-solidity.rst]: I think options does not refer to e.g. commandline options. I'll reword it.

**Review by ekpyron — APPROVED** [review]

**Comment by axic** [comment]
Oh wow these are nice clarifications. Some of the wording was really bad prior.

**Pull request merged**